cmd/vela-kubernetes/config.go

```diff
@@ -6,46 +6,74 @@ package main
 
 import (
 	"fmt"
-	"time"
+	"os/user"
+	"path/filepath"
+
+	"github.com/spf13/afero"
 
 	"github.com/sirupsen/logrus"
 )
 
-// Config represents the plugin configuration for Kubernetes config information.
+var appFS = afero.NewOsFs()
```

Comment:
just my opinion, but we could protect the initialization of this with `sync.once`
eg.
`
var (
    appFS whateverthatis
    setupOnce sync.Once
)

func InitAppFS() {
    appFS = afero.NewOsFs()
}

func GetAppFS() *whateverthatis {
	setupOnce.Do(InitAppFS)
	return appFS
}
`